server/api/game/controller.js

```diff
@@ -186,6 +186,16 @@ const calculateNextGameState = () => {
   gameState.ghosts = ghostsUpdated;
 };
 
+export const updateDirection = ({ playerId, direction }) => {
+  const { pacmanOne, pacmanTwo } = gameState;
+  if (pacmanOne.playerId === playerId) {
+    gameState.pacmanOne.direction = direction;
+  }
+  if (pacmanTwo.playerId === playerId) {
+    gameState.pacmanTwo.direction = direction;
+  }
+};
```

Comment:
2 pacman variables looks bad. What is the need to name and store in 2 different pacman variables? Use a `pacmans` object instead:

```
const pacmans = {
  playerId: { x, y, direction },
};```